modules/stochastic_tools/src/surrogates/PolynomialChaos.C

```diff
@@ -133,6 +57,30 @@ PolynomialChaos::evaluate(const std::vector<Real> & x) const
   return val;
 }
 
+const std::vector<std::vector<unsigned int>> &
+PolynomialChaos::getPolynomialOrders() const
+{
+  return _tuple;
+}
+
+unsigned
+PolynomialChaos::getPolynomialOrder(const unsigned int dim, const unsigned int i) const
```

Comment:
Minor: Usually you don't need to worry about marking pass-by-value parameters as const. You can't modify the caller context. There's also nothing wrong with it.

Reply:
Good point. I guess it only protects from modification within the function.

Reply:
I know it is pointless, but it prevents accidentally changing the value in the function. I also like to  show the user that the value being passed in isn't involved in the calculation.